src/twisted/trial/_synctest.py

```diff
@@ -43,31 +45,28 @@ class FailTest(AssertionError):
     """
 
 
+@attr.s(frozen=True, slots=True, auto_attribs=True)
 class Todo:
     """
-    Internal object used to mark a L{TestCase} as 'todo'. Tests marked 'todo'
+    Object used to mark a L{TestCase} as 'todo'. Tests marked 'todo'
```

Comment:
promoted to public so you can use `@unittest.mark(todo=Todo(reason="ham", errors=[Spam]))`